src/main/java/emu/grasscutter/data/ResourceLoader.java

```diff
@@ -269,7 +269,15 @@ private static void loadAbilityModifiers() {
 			List<AbilityConfigData> abilityConfigList;
 			
 			try (FileReader fileReader = new FileReader(file)) {
-				abilityConfigList = Grasscutter.getGsonFactory().fromJson(fileReader, TypeToken.getParameterized(Collection.class, AbilityConfigData.class).getType());
+                BufferedReader bufferedReader = new BufferedReader(fileReader);
+                String line = "", lines = "";
+                while((line = bufferedReader.readLine()) != null){
+                    lines += line;
+                }
+                lines = lines.replaceAll("HNEIEGHMLKH", "modifiers");
```

Comment:
It's recommended to use like `@SerializedName(value="x", alternate={"_x", "X"})` in data class instead, see `Position` class